tools/wt_verify/wt_verify.py

```diff
@@ -172,6 +237,129 @@ def parse_output():
     return output
 
 
+def show_page_distribution(filename, data, bar_width=1):
+    """
+    data: dictionary generated by parse_dump_blocks() that contains information related to different
+    checkpoints.
+    """
+    imgs = ""
+    bar_width = 1
+    colors = ['blue', 'orange', 'green', 'red', 'purple', 'brown', 'pink', 'gray', 'olive', 'cyan']
+    for checkpoint in data:
```

Comment:
The `for checkpoint in data` loop combined with the `imgs += img` line below reads like it's going to produce a separate chart for each checkpoint. Is this the case?

Reply:
Yes, it is the case.

Reply:
OK. I'm not sure how much sense that makes, since I tend to think of blocks at the file level rather than the checkpoint level. If that's hard to change then I guess we can make another ticket. Happy to discuss it offline, too.

Reply:
I agree with you, it was an easy change. See those three commits:

1. 144e4ea8a156b0c7cb694575599089f2816094fa
2. 3b23c38f4c85e40dce18ce87bbd8ea131373c9e8
3. 3e3b0cdb75fd9b3483d4686707c73cce29fd4077